tests/test_auth.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+
+from twine import (
+    auth,
+    exceptions,
+    utils,
+)
+
+
+cred = auth.CredentialInput.new
```

Comment:
Why is the `new` necessary? After a quick scan, it looks like `cred = auth.CredentialInput` would work.

Reply:
Ah:

```
E       TypeError: __new__() missing 1 required positional argument: 'password'
```

Which makes me wonder if `CredentialInput` should just be a class...